components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -15,149 +15,258 @@ import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 
 service Policies {
   rpc CreatePolicy (CreatePolicyReq) returns (CreatePolicyResp) {
-    option (google.api.http).post = "/iam/v2beta/policies";
-    option (google.api.http).body = "*";
+    option (google.api.http) = {
```

Comment:
Tried to look up the syntax for `option` in the language specification (https://developers.google.com/protocol-buffers/docs/reference/proto3-spec) but... it is missing!

However, a few paragraphs into https://cloud.google.com/endpoints/docs/grpc-service-config/reference/rpc/google.api#google.api.HttpRule, there are at least precedents for us to follow. Based on that, commas should not be used within `option` objects:

Mostly, you have correctly omitted commas, but there are some lurking that should be deleted:
L22
L35
L59
L72
L110
L124
L138
L152
L202
L216
L230

(Similar corrections needed in other proto files.)